apps/explorer/src/components/checkpoints/CheckpointsTable.tsx

```diff
@@ -39,13 +39,20 @@ export function CheckpointsTable({
         rpc.getLatestCheckpointSequenceNumber()
     );
 
+    const checkpoints = useGetCheckpoints(initialCursor, limit);
+
     const count = useMemo(() => {
-        if (maxCursor && initialCursor)
+        if (maxCursor && initialCursor) {
             return Number(initialCursor) - Number(maxCursor);
+            // Special case for ongoing epoch
```

Comment:
super nit: this might be just me, but I got confused looking at the comment because it's in the block for the non-special case 😆. Could we maybe move this in between line 48 and 49? also elaborating a little bit might be helpful since I'm not really sure why we need the special case or why the first sequenceNumber - the max cursor is the correct count

Reply:
[slack thread for context](https://mysten-labs.slack.com/archives/C032676BWGN/p1683239640505209)

Reply:
thanks! would be great to put that into a comment for future maintainers since there's a lot going on in Slack and not sure if the history is permanent :)